test/testBuildOutput.js

```diff
@@ -0,0 +1,70 @@
+var expect = require('chai').expect;
+var fs = require('fs');
+
+describe('Build process', function() {
+  before(function () {
+  });
+
+  it('should output readable expected files and only expected files', function (done) {
+    // Please keep this list alphabetically sorted and lowercase
+    let expectedFiles = [ 'bundle.css', 'bundle.css.map', 'bundle.js', 'bundle.js.map', 'index.html'];
+
+    fs.readdir('dist', function (err, files) {
+      if (err) {
+        done(err);
+      }
+
+      let promises = [];
+      files.forEach(function (filename) {
+        promises.push(processExistingFile(filename));
+      });
+
+      Promise.all(promises).then(function () {
+        if (0 !== expectedFiles.length) {
```

Comment:
No yoda conditions please. I was hoping the linter would catch that though, sorry.